app/cypress/integration/email.spec.js

```diff
@@ -133,44 +133,6 @@ describe("Confirmation emails", () => {
   beforeEach(() => {
     cy.mockLogin("reporter");
   });
-
```

Comment:
This is the only test in this describe block so I think we can drop the whole thing